style.css

```diff
@@ -1 +1,270 @@
 /*  Add your styling here */
+/* styling header section */
+body{
+    margin:0px;
+    font-family: 'Langar', cursive;
+}
+header{
```

Comment:
Type selector `header` is very generic. `<header>` tag may be used multiple times in an html document (eg inside an `<article>`) which means that the styles you write here will be applied on all headers you may add on the page. Use more specific selectors like class selector to avoid confusion. A class selector will also be arguably more efficient. Check this out: https://csswizardry.com/2011/09/writing-efficient-css-selectors/ 😉